### ISSUE TITLE
Bazel does not fail for empty command

### ISSUE THREAD

**Issue by avdv** [bazelbuild/bazel#24577]
### Description of the bug:

When running `bazel '' test ...` (note the empty argument) Bazel just displays its default help text (as if just running `bazel`) and exits with code `0`.

When using an invalid command like `bazel doflips ...` it correctly reports an error and exits with code `2`.

Context: I got bitten by this when the empty argument sneaked into the command line, CI was green when it actually wasn't.

### Which category does this issue belong to?

CLI

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

Run `bazel '' test ...`

### Which operating system are you running Bazel on?

Windows, Linux

### What is the output of `bazel info release`?

5.4.1 - 8.0.0rc7

**Comment by iancha1992** [comment]
A fix for this issue has been included in [Bazel 8.1.0 RC1](https://github.com/bazelbuild/bazel/releases/tag/8.1.0rc1). Please test out the release candidate and report any issues as soon as possible.
If you're using Bazelisk, you can point to the latest RC by setting USE_BAZEL_VERSION=8.1.0rc1. Thanks!

**Comment by avdv** [comment]
Confirming that this works in 8.1.0RC1:
```
USE_BAZEL_VERSION=8.1.0rc1 bazel '' build ...
[FATAL 11:41:07.153 src/main/cpp/blaze.cc:1194] Command cannot be the empty string.
  For more info, run 'bazel help'.
```
